src/main/resources/db/changelog/db.changelog-1.0.xml

```diff
@@ -0,0 +1,251 @@
+            <column name="creation_time" type="TIMESTAMP WITH TIME ZONE" defaultValueComputed="CURRENT_TIMESTAMP">
+                <constraints nullable="false"/>
+            </column>
+            <column name="modification_time" type="TIMESTAMP WITH TIME ZONE" defaultValueComputed="CURRENT_TIMESTAMP">
+                <constraints nullable="false"/>
+            </column>
+            <column name="deletion_time" type="TIMESTAMP WITH TIME ZONE"/>
+
+            <column name="authorisation_id" type="BIGINT">
+                <constraints nullable="false"/>
+            </column>
+            <column name="begin_date" type="DATE">
+                <constraints nullable="false"/>
+            </column>
+            <column name="end_date" type="DATE"/>
+        </createTable>
+        <addForeignKeyConstraint baseTableName="validity" baseColumnNames="authorisation_id"
+                                 referencedTableName="authorisation" referencedColumnNames="authorisation_id"
+                                 constraintName="fk_validity_authorisation"
+                                 onDelete="NO ACTION" onUpdate="NO ACTION"/>
+        <sql><![CDATA[
+            ALTER TABLE validity
+                ADD CONSTRAINT ck_validity_end_date
+                    CHECK (end_date IS NULL OR begin_date < end_date)
+            ]]></sql>
+        <!--TODO create trigger which checks that if end date is null then authorisation.justification_name='VIR' ? -->
+    </changeSet>
+
+    <changeSet id="2021-11-02-create-validity_reminder-table" author="terova">
+        <createTable tableName="validity_reminder">
+            <column autoIncrement="true" name="validity_reminder_id" type="BIGSERIAL">
+                <constraints primaryKey="true" primaryKeyName="validity_reminder_pkey"/>
+            </column>
+            <column name="version" type="INT" defaultValueNumeric="0">
+                <constraints nullable="false"/>
+            </column>
+            <column name="created_by" type="TEXT"/>
+            <column name="deleted_by" type="TEXT"/>
+            <column name="modified_by" type="TEXT"/>
+            <column name="creation_time" type="TIMESTAMP WITH TIME ZONE" defaultValueComputed="CURRENT_TIMESTAMP">
+                <constraints nullable="false"/>
+            </column>
+            <column name="modification_time" type="TIMESTAMP WITH TIME ZONE" defaultValueComputed="CURRENT_TIMESTAMP">
+                <constraints nullable="false"/>
+            </column>
+            <column name="deletion_time" type="TIMESTAMP WITH TIME ZONE"/>
+
+            <column name="validity_id" type="BIGINT">
+                <constraints nullable="false"/>
+            </column>
+            <column name="from" type="text">
+                <constraints nullable="false"/>
+            </column>
+            <column name="to" type="text">
+                <constraints nullable="false"/>
+            </column>
```

Comment:
from ja to ovat varattuja sanoja, vaihdan ne sender ja recipient, jos joutuu kirjoittamaan mitään SQL kyselyjä käsin on niiden kirjoittaminen helpompaa.